docs/source/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/editingCode.rst

```diff
@@ -170,6 +171,14 @@ Some useful commands::
 and specifically `miniconda <https://conda.io/en/latest/miniconda.html>`_.
 Use it if you like, if you understand virtualenv you can understand conda environments.
 
+Some useful conda commands::
```

Comment:
I propose to change the conda examples to:
```suggestion
Some useful conda commands::

  conda env create $DEVROOT/DIRAC/environment.yml  # will create a conda environment named
                                                   # dirac-development and install the
						   # prerequiste packages

  conda activate dirac-development #  will get you in the dirac-development environment
  # or on conda versions prior to 4.6
  source activate dirac-development

  conda deactivate  # will exit from the conda environment
  # or on conda versions prior to 4.6 
  source deactivate
```
This avoids having to scroll to read everything and makes it easier to copy individual commands into a shell